Makefile

```diff
@@ -26,11 +25,35 @@ qa: bin
 	scripts/qa/vet-high-confidence.sh
 	# qa-post-build steps minus stack tests
 	scripts/qa/docs.sh
+
+#debug versions for remote debugging with delve
+bin-debug:
+	./scripts/build.sh -debug
+docker-debug:
+	./scripts/build_docker.sh -debug
+qa-debug: bin-debug
```

Comment:
let's isolate out all the qa commands so we don't have the repetition. (e.g. into a qa-common target)
you can then make qa-debug and qa depend on qa-common